docs/en/advanced-concepts/migrating-to-appium-2.0.md

```diff
@@ -127,3 +127,81 @@ TODO
 
 TODO
 
+## Special Notes for Cloud Providers
+
+The rest of this document has applied to Appium generally, but some of the architectural changes in Appium 2.0 will constitute breaking changes for Appium-related service providers, whether a cloud-based Appium host or an internal service. At the end of the day, the maintainer of the Appium server is responsible for installing and making available the various Appium drivers and plugins that end users may wish to use.
+
+With Appium 2.0, we enter a new era where end users may wish to target various independent versions of drivers and plugins. With Appium 1.x, this was never the case, since any given version of Appium would contain one and only one version of each driver. It is of course up to each service provider how they wish to implement the discovery, installation, and availability of any official or third party drivers or plugins. But the Appium team wants to make a recommendation in terms of the capabilities service providers support, for consistency across the industry. This is a recommendation only, and not a standard, but adopting it will help users to navigate the increased complexity that working with Appium 2.0 in a cloud environment may bring.
+
+### Suggested capabilities
+
+In addition to the standard `platformName`, `appium:deviceName`, `appium:automationName`, and `appium:platformVersion`, we recommend adopting the following set:
```

Comment:
What about replacing `appium:` with `<your vendor prefix>:` (or similar) to clarify they are not appium capabilities?

Appium expects `appium:` prefix capabilities are for Appium, but they are not for Appium capabilities. They will always print as "not appium capabilities" warning message in a new session request log in appium. This could be noise. (not issue tho)

I guess this main reason is to standardized appium stuff for end-users to reduce confusion.
So, like...:

`<your vendor prefix>:appiumVersion`
`<your vendor prefix>:appiumAutomationVersion`  (I wondered "driverVersion", but mm, actually "automationVersion" is probably better naming for "automationName".
`<your vendor prefix>:appiumAutomation`
`<your vendor prefix>:appiumPlugins`

Reply:
yeah I think that makes sense to disambiguate it from the things that appium itself uses. it does mean each cloud would have a separate set but i guess that's OK because they are in fact different vendors.